core/controllers/admin_test.py

```diff
@@ -161,7 +169,274 @@ def test_cannot_reload_exploration_in_production_mode(self):
 
         self.logout()
 
-    def test_cannot_load_new_structures_data_in_production_mode(self):
+    def test_without_exp_id_reload_exp_action_is_not_performed(self) -> None:
+        self.login(self.CURRICULUM_ADMIN_EMAIL, is_super_admin=True)
+        csrf_token = self.get_new_csrf_token()
+
+        prod_mode_swap = self.swap(constants, 'DEV_MODE', False)
```

Comment:
You can maybe put this into `setUp`, so you don't need to redefine it in every function.

Reply:
Done!